src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -716,6 +716,29 @@ protected boolean doubleClick(WebElement element) {
         return result;
     }
 
+    @WaitUntil
+    public boolean clickWithOffsetXY(String place,int xOffset,int yOffset) {
+        return  clickWithOffsetXYIn(place, xOffset, yOffset, null);
+    }
+
+    @WaitUntil
+    public boolean clickWithOffsetXYIn(String place,int xOffset,int yOffset, String container) {
+        WebElement element = getElementToClick(place, container);
+        return clickWithOffsetXY(element, xOffset, yOffset);
+    }
+
+    protected boolean clickWithOffsetXY(WebElement element,int xOffset,int yOffset) {
+        boolean result = false;
+        if (element != null) {
+            scrollIfNotOnScreen(element);
```

Comment:
Should we not scroll to the location where we are going to click? Maybe first scroll to the element, and then scroll with the offset?